Source/Extensions/Blazorise.Charts/ChartColor.cs

```diff
@@ -40,8 +41,123 @@ public ChartColor( float red, float green, float blue, float alpha )
 
     #endregion
 
+    #region Operators
+
+    /// <summary>
+    /// Implicitly converts from string value to <see cref="ChartColor"/>.
+    /// </summary>
+    /// <param name="value">String value.</param>
+    public static implicit operator ChartColor( string value )
+    {
+        if ( value.StartsWith( '#' ) )
+            return HexStringToColor( value );
+        else if ( value.StartsWith( "rgb" ) )
+            return CssRgbaFunctionToColor( value );
+
+        throw new FormatException( $"Invalid chart color format: {value}" );
+    }
+
+    #endregion
+
     #region Methods
 
+    /// <summary>
+    /// Checks for characters that are Hexadecimal
+    /// </summary>
+    private static Regex IsHexDigit = new( "[abcdefABCDEF\\d]+", RegexOptions.Compiled );
+
+    /// <summary>
+    /// Extract only the hex digits from a string.
+    /// </summary>
+    /// <param name="input">A string to extract.</param>
+    /// <returns>A new hex string.</returns>
+    private static string ExtractHexDigits( string input )
+    {
+        var sb = new StringBuilder();
+        var result = IsHexDigit.Matches( input );
+
+        foreach ( System.Text.RegularExpressions.Match item in result )
+        {
+            sb.Append( item.Value );
+        }
+
+        return sb.ToString();
+    }
```

Comment:
Isn't there somewhere else with this code? Can it be shared?

Reply:
There is in ThemeGenerator. Didn't want to make it public so and was easy to just copy/paste.